go.mod

```diff
@@ -35,6 +37,8 @@ require (
 	github.com/multiformats/go-multiaddr v0.4.0
 	github.com/multiformats/go-multihash v0.0.15
 	github.com/stretchr/testify v1.7.1-0.20210427113832-6241f9ab9942
+	github.com/tendermint/tendermint v0.34.13
```

Comment:
Meh, I wish the tendermint protos would live in another repository so we wouldn't need to import the whole repo.
Would it not make sense here to import celestia-core's proto files here? That one is already a dependency. Also, the module name changed on master and will soon be tendermint again: 
https://github.com/celestiaorg/celestia-core/blob/c2df2c4ce0942b8e974fd2b530355f2a58139926/go.mod#L1

Reply:
I think we need to import tendermint and add a replace directive here for celestia-core. Not only for consistency sake, but because that's how celestia-core declare's its path. I think we also need be using the same tagged version of celestia-core as the app uses (v0.34.12-celestia)